VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/vh-officer-venue-list/vh-officer-venue-list.component.ts

```diff
@@ -117,20 +106,20 @@ export class VhOfficerVenueListComponent extends VenueListComponentDirective imp
         }
     }
 
-    private getFiltersCourtRoomsAccounts(response: CourtRoomsAccountResponse[]) {
-        const updateFilterSelection = (filterVenue: CourtRoomsAccounts) => {
-            const courtroomAccount = this.filterCourtRoomsAccounts.find(x => x.venue === filterVenue.venue);
-            if (courtroomAccount) {
-                courtroomAccount.selected = filterVenue.selected;
-                courtroomAccount.updateRoomSelection(filterVenue.courtsRooms);
-            }
-        };
-        this.filterCourtRoomsAccounts = response.map(x => new CourtRoomsAccounts(x.venue, x.rooms, true));
-        const previousFilter = this.courtAccountsAllocationStorage.get();
-        if (previousFilter) {
-            previousFilter.forEach(x => updateFilterSelection(x));
-        }
-        this.courtAccountsAllocationStorage.set(this.filterCourtRoomsAccounts);
-        this.logger.debug('[VenueList] - Venue selection is changed');
-    }
+    // private getFiltersCourtRoomsAccounts(response: CourtRoomsAccountResponse[]) {
```

Comment:
some commented out code here

Reply:
ah great shout! thank you

Reply:
I'll add some coverage since this code is now in the pipe